diesel_cli/src/main.rs

```diff
@@ -460,6 +460,14 @@ fn run_infer_schema(matches: &ArgMatches) -> Result<(), Box<dyn Error>> {
         config.with_docs = true;
     }
 
+    if let Some(sorting) = matches.value_of("column-sorting") {
+        match sorting {
+            "ordinal_position" => config.column_sorting = ColumnSorting::OrdinalPosition,
+            "name" => config.column_sorting = ColumnSorting::Name,
+            _ => unreachable!("Invalid column sorting mode: {}", sorting),
+        }
+    }
```

Comment:
I tried using `arg_enum!`, but this macro does not support doc comments and `#[derive(Debug, Deserialize, Serialize)]`.
https://docs.rs/clap/2.33.0/clap/macro.arg_enum.html

Reply:
It has been removed in clap v3 master because of bugs like this. We implemented ArgEnum derive. 

But yeah, this is good enough for now.

Reply:
Thank you for the information! That's news to me.